services/app-api/handlers/reports/lock.test.ts

```diff
@@ -0,0 +1,95 @@
+import { fetchReport } from "./fetch";
```

Comment:
I had planned to handle this differently—instead of a discrete lock route, handling this change in the submit handler. Since admins only unlock, locking is just the MLR version of submitting (in my mind). Thoughts?

Reply:
That's a good point. I'll remove the `lock` handler from this PR and we can handle it under submit going forward.